include/signalrclient/hub_connection_builder.h

```diff
@@ -42,5 +44,6 @@ namespace signalr
         trace_level m_log_level;
         std::function<std::shared_ptr<websocket_client>(const signalr_client_config&)> m_websocket_factory;
         std::shared_ptr<http_client> m_http_client;
+        bool m_skip_negotiation;
```

Comment:
This is currently uninitialized which is causing errors
```suggestion
        bool m_skip_negotiation = false;
```